src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -248,7 +248,10 @@ public static void ImportNoExponent()
 
             using (RSA rsa = RSAFactory.Create())
             {
-                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                if (rsa is RSACryptoServiceProvider)
+                    Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                else
+                    Assert.ThrowsAny<ArgumentException>(() => rsa.ImportParameters(imported));
```

Comment:
They are failing. I wasn't sure if we wanted RSAOpenSsl to behave like RSACng or RSACryptoServiceProvider i.e. throw ArgumentException or CryptoException. Seems like ArgumentException would be preferable, but it's not something we've discussed.